source/_integrations/slow_pwm.markdown

```diff
@@ -0,0 +1,113 @@
+---
+title: Slow PWM
+description: Convert one or multiple switches to a numeric input using slow Pulse-Width-Modulation
+ha_category:
+  - Number
+ha_release: '2023.05'
+ha_config_flow: true
+ha_iot_class: Local Polling
+ha_codeowners:
+  - '@antonverburg'
+ha_domain: slow_pwm
+ha_platforms:
+  - number
+ha_integration_type: integration
+---
+
+The `slow_pwm` platform is a `number`. It can be used to control one or multiple digital switches in an analog control algorithm. 
+The switches will be controlled by Pulse-Width-Manipulation (PWM, See https://en.wikipedia.org/wiki/Pulse-width_modulation). 
```

Comment:
```suggestion
The switches will be controlled by pulse width modulation (PWM, see https://en.wikipedia.org/wiki/Pulse-width_modulation). 
```
I realize that Wikipedia spells it differently. I went for the spelling used by [Marriam Webster](https://www.merriam-webster.com/dictionary/pulse%20width%20modulation), which I find to be a reliable resource.